src/core/audio/AudioFileOgg.cpp

```diff
@@ -136,8 +138,13 @@ bool AudioFileOgg::startEncoding()
 
 	// We give our ogg file a random serial number and avoid
 	// 0 and UINT32_MAX which can get you into trouble.
-	qsrand( time( 0 ) );
+#if (QT_VERSION >= QT_VERSION_CHECK(5,10,0))
+	QRandomGenerator::global()->seed(time(0));
+	m_serialNo = 0xD0000000 + QRandomGenerator::global()->generate() % 0x0FFFFFFF;
```

Comment:
Reported on Discord: an user got the following error on OGG export.
> Attempted to overwrite a QRandomGenerator to system() or global().

The documentation says `QRandomGenerator::global()` is already seeded and can't be overwritten by `seed()`.

Reply:
Open a new issue with this information so it gets attention and a PR can reference it?